### PR TITLE
AV-429 Added ability to allow iframe

### PR DESCRIPTION
**Pull request by ilyaGotfryd**

### CODE REVIEW

**Review by aporlando — APPROVED** [review]

File: run_nginx.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+docker run -it \
```
Comment (lieutdan13) [inline run_nginx.sh]: Let's remove this file. Keep it locally if you need it.

File: start.sh [hunk 2]
```diff
@@ -52,5 +52,15 @@ fi
 # Tell nginx the address and port of the service to proxy to
 sed -i "s/{{TARGET_SERVICE}}/${TARGET_SERVICE}/g;" /etc/nginx/conf.d/proxy.conf
 
+# Tell nginx if you want to block or allow iframing of pages and what domain name is allowed to iframe your content
+if [ -n "${XFRAME_OPTION+1}" ]; then
+  echo "Enabling iframeing..."
```
Comment (lieutdan13) [inline start.sh:57]: You can remove these echos

File: start.sh [hunk 3]
```diff
@@ -52,5 +52,15 @@ fi
...
+  sed -i "s/{{XFRAME_VALUE}}/${XFRAME_OPTION}/g;" /etc/nginx/conf.d/proxy.conf
+else
+  echo "Disabling iframeing..."
```
Comment (lieutdan13) [inline start.sh:60]: Remove the echo

**Comment by lieutdan13** [comment]
You should also commit --amend to hide the onshift specific changes you are removing

**Comment by ilyaGotfryd** [comment]
@lieutdan13 I have made a change per your request.

**Review by lieutdan13 — CHANGES_REQUESTED** [review]

**Comment by lieutdan13** [comment]
@ilyaGotfryd sorry. I did not finish my CR from this morning.

**Comment by lieutdan13** [comment]
Also, @ryanhaffey has some concerns with dependency management. We are making custom changes and now relying on an external dependency.

File: start.sh (hunk 2 above)
Comment (ilyaGotfryd) [reply start.sh:57]: I followed the pattern they had above, but I'll be happy about it.

Comment (lieutdan13) [reply start.sh:57]: I see. Then I would change it to "Applying iframe option..."

File: start.sh (hunk 3 above)
Comment (lieutdan13) [reply start.sh:60]: I would change this to "Denying iframes..."

**Pull request closed without merging**